dell/latitude/3340/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
```

Comment:
This profile usually already provides sane defaults for the builtin graphic card.